### ISSUE TITLE
abort with status_code=401

### ISSUE THREAD

**Issue by python-programmer** [channelcat/sanic#1030]
when i using abort with 401 status code:

```
@app.get('/api/abort/with/message')
async def abort_with_message(request):
    abort(401, message='unauthorized')
    return text('authorized')
```

 i get this error:

```
raise sanic_exception(message=message, status_code=status_code)
TypeError: __init__() missing 1 required positional argument: 'scheme'
```

**Comment by ashleysommer** [comment]
Duplicate of #914
Fixed in #917 
Use sanic @master if you need the fix _now_, otherwise wait for the 0.7.0 release soon.

**Comment by python-programmer** [comment]
thank you for reply
